include/server/api/base_parameters_grammar.hpp

```diff
@@ -111,15 +111,15 @@ struct BaseParametersGrammar : boost::spirit::qi::grammar<Iterator, Signature>
                                                 qi::_1,
                                                 qi::_2)];
 
-        location_rule =
-            (double_ > qi::lit(',') >
-             double_)[qi::_val = ph::bind(
-                          [](double lon, double lat) {
-                              return util::Coordinate(util::toFixed(util::FloatLongitude{lon}),
-                                                      util::toFixed(util::FloatLatitude{lat}));
-                          },
-                          qi::_1,
-                          qi::_2)];
+        location_rule = (double_ > qi::lit(',') >
+                         double_)[qi::_val = ph::bind(
+                                      [](double lon, double lat) {
+                                          return util::Coordinate(
+                                              util::toFixed(util::UnsafeFloatLongitude{lon}),
```

Comment:
Here's an example for a `RealPolicy` we're already using

https://github.com/Project-OSRM/osrm-backend/blob/b1f6797aab453e141931af2389b8e16687e47fd6/include/server/api/base_parameters_grammar.hpp#L30-L68

see for context https://github.com/Project-OSRM/osrm-backend/issues/2009